code/modules/hydroponics/trays/tray.dm

```diff
@@ -19,7 +19,7 @@
 	var/maxWaterLevel = 100
 	var/maxNutriLevel = 10
 	var/maxPestLevel = 10
-	var/maxWeedLevel = 10
+	var/maxWeedLevel = 0 // Hydroponics systems don't grow weeds irl. Pests are still an issue.
```

Comment:
I feel that you're removing depth to gameplay when you make weeds as a non-factor for hydroponics trays.

Reply:
How so?